Source/Core/Core/PowerPC/Jit64/Jit.cpp

```diff
@@ -158,27 +163,55 @@ Jit64::Jit64() : QuantizedMemoryRoutines(*this)
 
 Jit64::~Jit64() = default;
 
-void Jit64::AllocStack()
+void Jit64::ProtectStack()
```

Comment:
This method is mostly shared with the arm jit, so what do you think about moving the initialization to Common/Thread.cpp?

Reply:
Even better, thanks.